commit bee7369d9445219aed06816b0d07ccbf73dd2942
Author: Venkatesh Perumal <quic_venper@quicinc.com>
Date:   Tue Jun 25 12:58:51 2024 +0530

    generator: nanopb_generator.py Yocto upgrade (nanbield)

    Compilation fixes for following Yocto versions.
    nanbield and scarthgap versions.

Upstream-Status: Inappropriate [oe specific]

    Signed-off-by: Venkatesh Perumal <quic_venper@quicinc.com>

diff --git a/generator/nanopb_generator.py b/generator/nanopb_generator.py
index e4d3d27..e971f43 100755
--- a/generator/nanopb_generator.py
+++ b/generator/nanopb_generator.py
@@ -1502,7 +1502,7 @@ def parse_file(filename, fdesc, options):
             optfilename = os.path.join(p, optfilename)
             if options.verbose:
                 sys.stderr.write('Reading options from ' + optfilename + '\n')
-            Globals.separate_options = read_options_file(open(optfilename, "rU"))
+            Globals.separate_options = read_options_file(open(optfilename, "r"))
             break
     else:
         # If we are given a full filename and it does not exist, give an error.
